src/code/templates/genome-playground.js

```diff
@@ -18,13 +18,15 @@ export default class GenomeContainer extends Component {
     let penView = <div className='columns bottom'>
                     <PenView orgs={ keptDrakes } width={500} columns={5} rows={1} tightenRows={20}/>
                   </div>;
-    
+
+    // we don't increment the score, so users always get top mark from playground
+    const kNoScoreIncrement = false;
```

Comment:
I'm confused by this variable name- why choose something other than `incrementMoves`?